crates/object_detection/src/pose_detection.rs

```diff
@@ -89,8 +89,10 @@ impl PoseDetection {
                 weights_path
                     .to_str()
                     .wrap_err("failed to get detection weights path")?,
-            )
-            .wrap_err("failed to create detection network")?;
+            ).map_err(|error| match error{
+                openvino::InferenceError::GeneralError => eyre!("General Error: Please ensure that you have a complete & working OpenVino installation."),
```

Comment:
Please replace qualified path with use. This error message is different than our usual error messages, don't you think the error message should just state the error, not propose solutions? Please also make the error message lowercase

Reply:
Yes you are spot on about `use`, didn't notice it at first.
I wasn't aware of the lower case convention, will change.

As for the error message, they use GeneralError in a rather ambiguous way, could be reformulated to say "possible problem with missing OpenVino components" as this is one of the reasons.

As for errors with suggestions: when they are directly shown to the user or closer to the user, I feel it might make more sense to give a hint than just say we failed.